scaper/core.py

```diff
@@ -1966,16 +1964,23 @@ def _generate_audio(self,
                     soundscape_audio, event_audio_list, scale_factor = \
                         peak_normalize(soundscape_audio, event_audio_list)
 
-                    warnings.warn(
-                        'Peak normalization applied (scale factor = {})'.format(
-                            scale_factor),
-                        ScaperWarning)
+                    ref_db_drop = 20 * np.log10(scale_factor)
```

Comment:
I'd say `ref_db_change` instead of `ref_db_drop`, as if `scale_factor > 1.0`, then it's actually a gain instead of a drop.

Reply:
haha didn't see this comment till now but I had the very same thought and changed it!